configure.py

```diff
@@ -470,6 +470,7 @@ def find_headers(repodir, excluded_dirs):
                 'table.cc',
                 'atomic_cell.cc',
```

Comment:
Please split into two patches: infrastructure up to the new API in connection_notifier.hh, and a second patch that uses the API in transport/server.cc. This eases review by allowing the reviewer to grok the infrastructure, and then consider how it is used.

Reply:
(and if we implement thrift and alternator support, these would come as further separate patches that can be reviewed by those protocols' maintainers)

Reply:
Done